src/GraphQL/Execution/ExecutionError.cs

```diff
@@ -26,15 +29,36 @@ public ExecutionError(string message, IDictionary data)
         public ExecutionError(string message, Exception exception)
             : base(message, exception)
         {
-            var ex = exception?.InnerException ?? exception;
```

Comment:
I believe that it would be more correct to take the code from the exception and not try to look further

Reply:
agreed